Gruntfile.js

```diff
@@ -163,7 +163,6 @@ module.exports = function (grunt) {
 
     cssmin: {
       options: {
-        compatibility: 'ie10,-properties.zeroUnits',
```

Comment:
`ie10` is not a valid clean-css option, and `-properties.zeroUnits` is no-longer needed now we're removing the relevant browser hack.